pytentiostat/config_reader.py

```diff
@@ -118,6 +111,6 @@ def get_adv_params(adv_config_data):
 
 if __name__ == "__main__":
     # used for debugging.  Does the function load the configs?
-    data, adv_data = parse_config_files("pytentiostat/tests/static")
+    data = parse_config_files("pytentiostat/tests/static")
```

Comment:
singular?

Reply:
forgot at first but fixed in later commit

Reply:
"Then we should think of what other things we want to test for this guy."

Yeah I'll think about some other things to make more robust.